Inventory.cs

```diff
@@ -0,0 +1,82 @@
+﻿public class Inventory
+{
+    private List<Product> products = new();
+
+    public void Add(string name, decimal price, int quantity)
+    {
+        var newProduct = new Product(name, price, quantity);
+        if (Product.AreValid(price, quantity)) 
```

Comment:
It's a bit confusing to pass product values and validate them, the idea is that Product class already have the values and you can just add methods to perform on those values. So maybe just do `newProduct.IsValid()` instead ? And would be even better if you have this type of validations in a different extension class `ProductValidator`

Reply:
If the values are not valid, here you need to throw an exception that `Main` should catch. It's the `Main` responsibility to do I/O. We shouldn't have print statements in Product nor Inventory classes, since you can use those classes for a 
different project such as WebAPI not Console App that doesn't understand much the print statements. 
Print statements are specific to the framework (console app), and you don't want to couple your domain with the framework, so you can easily replace framework later.